source/examples/get_devices.out

```diff
@@ -0,0 +1,5 @@
+# SPDX-FileCopyrightText: 2020 The Khronos Group Inc.
```

Comment:
```suggestion
# SPDX-FileCopyrightText: 2023 The Khronos Group Inc.
```

Reply:
Updated it here. Should we update copyright text for all changed files in the future, or only for newly created ones?

Reply:
We'll discuss this in the SYCL WG call and get back to you